otter/convergence/service.py

```diff
@@ -29,22 +99,24 @@ def _get_lock(self, group_id):
         lock.acquire = partial(lock.acquire, timeout=120)
         return lock
 
-    def start_convergence(self, log, tenant_id, group_id, desired,
+    def start_convergence(self, log, scaling_group, group_state,
                           launch_config,
                           perform=perform,
                           execute_convergence=execute_convergence):
         """Converge a group to a capacity with a launch config."""
```

Comment:
If you don't mind, could you add types for params docstring? Especially for `scaling_group` and `group_state` as they can be confusing. I realize they should probably not be there in the first place :/